.idea/jarRepositories.xml

```diff
@@ -16,6 +16,11 @@
       <option name="name" value="arim-mvn-lgpl3" />
       <option name="url" value="https://mvn-repo.arim.space/lesser-gpl3/" />
     </remote-repository>
+    <remote-repository>
```

Comment:
I suggest adding .idea/ to your .gitignore

Reply:
too late for it now haha